### DISCUSSION TITLE
Adding salary expenses to Frappe books.

### DISCUSSION THREAD

**Discussion by njmulsqb** [frappe/books#730] [Q&A]
Hi,

I am facing hard time to figure out how can I add salary expense to the books? When I open chart of accounts, under indirect expenses I can see the salary category, but when I go to purchase payments and create a payment there, under Party --> Default Account, creditors is selected by default, I want to replace this with Salary so that I know that this is a salary expense but cant seem to find salary here. 

Also, where can I give title to the expenses?

**Comment by njmulsqb** [comment]
When I try to create a Salary account it says Account already exists, but it is only showing creditors account in the dropdown
![image](https://github.com/frappe/books/assets/47940898/5c220f30-b82d-419e-8ddc-a9c59d27ec39)
![image](https://github.com/frappe/books/assets/47940898/7b5078bd-1a8d-4823-89ed-955755d05d78)

**Comment by 18alantom** [comment]
![Screenshot 2023-09-12 at 15 32 23](https://github.com/frappe/books/assets/29507195/52376e78-f9ba-43d8-8620-abe7df1d7779)

The salary account would be set on the Item you select when you create a Purchase Invoice, payments will be recorded when you submit this invoice and create a payment against it. ([docs](https://docs.frappebooks.com/transactions/purchase-invoices.html#making-purchase-payments), note the screenshots are slightly outdated in the docs)

**Comment by njmulsqb** [comment]
Right, also if you can tell me where can I see the current balance after deduction of all expenses? If its only possible through Chart of Accounts, isnt it too inside the pages. The dashboard should display the current balance/cash holding of my business shouldn't it?

**Reply by 18alantom** [reply]
If you are checking the Income after deduction of Expenses then you can check the [Profit and Loss report](https://docs.frappebooks.com/analytics/profit-and-loss.html). Else the Trial Balance will give you an idea of balances across all the accounts.

**Reply by njmulsqb** [reply]
Trial Balance is showing incorrect balance; the correct balance is only visible from Chart of Accounts section.

**Reply by 18alantom** [reply]
Can you tell me what's expected and what it's showing, steps to reproduce, etc. It must be a bug.

**Reply by njmulsqb** [reply]
Expected is 70,000 and showing is almost double. Steps to reproduce? Nothing much just added some entries in journal, no entries of invoices

**Comment by Sunitsarode** [comment]
I think the problem is quite different..
When try to create a Salary account, it says Account already exists, but it is only showing creditors account in the dropdown. 
Why we keep this restriction to the Dropdown menu.. The account dropdown should show all exist accounts (Including Salary and all others).  Is it correct?
